package/src/components/QuantityInput/v1/QuantityInput.js

```diff
@@ -123,6 +123,8 @@ class QuantityInput extends Component {
                 variant="outlined"
                 onClick={this.handleDecrementButton}
                 className={incrementButton}
+                disableRipple={true}
```

Comment:
since these are boolean props you can just add the prop name `<ButtonBase disableRipple ... />` no need for the `={true}`